src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,58 @@
+main {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  margin: 40px;
+  padding: $calendarPadding;
+  flex-wrap: wrap;
+  gap: $dayGap;
+  max-width: $weekWidth;
+
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+  
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include square(100px);
+    background-color: $dayColor;
+    border: 1px solid #000;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hoverColor;
+      transform: translate(0, -20px);
+      transition: 0.5s;
```

Comment:
indicate transition in &__day, not in hover. Then transition will work in both ways (when block with day goes up and down)